File: eval/fixtures/php_demo/DiagnosticsController.php
<?php

namespace App\Http\Controllers;

use App\Services\CommandService;

class DiagnosticsController
{
    #[Route("/ping", methods: ["GET"])]
    public function ping($host)
    {
        return CommandService::runPing($host);
    }

    #[Route("/ping/safe", methods: ["GET"])]
    public function pingSafe($host)
    {
        return CommandService::runPingSafe($host);
    }
}

File: eval/fixtures/scala_demo/app/controllers/DiagnosticsController.scala
package controllers

import services.CommandService

class DiagnosticsController {
  def ping(host: String): String = {
    CommandService.runPing(host)
    "ok"
  }

  def pingSafe(host: String): String = {
    CommandService.runPingSafe(host)
    "ok"
  }
}
